cfpb/consumerfinance.gov
# Update design-system 0.27.2

Contains
https://github.com/cfpb/design-system/pull/1665

## Changes

- Move `-webkit-font-smoothing: antialiased;` from cfgov to design-system.


## How to test this PR

1. Run a local server and inspect the page `<body>` element and see that CSS rule `-webkit-font-smoothing: antialiased;` is still applied.
